.github/workflows/ci-jest-coverage.yml

```diff
@@ -0,0 +1,15 @@
+name: 'coverage'
+on:
+    pull_request:
+        branches:
+            - master
+            - main
+jobs:
+    coverage:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - uses: ArtiomTr/jest-coverage-report-action@v2
```

Comment:
I see in their docs that it can "Comparing coverage with base branch". Were you able to test that out or is it dependent on something existing on the base branch first?

Reply:
Yup, I think we've got that working, though it's difficult to tell in this actual PR since it isn't changing any files that would be covered. As [mentioned in this discussion](https://github.com/ArtiomTr/jest-coverage-report-action/discussions/96), apparently the spoiler tags w/ more info (& comparisons) only show up if there are relevant files that are changed. 

I could add some sort of TS file here in this PR for testing, but I'm more inclined to just try merging this and see how it works on other PRs. Open to other thoughts though!